packages/react-data-grid/src/SummaryRowRenderer.tsx

```diff
@@ -4,28 +4,28 @@ import Row from './Row';
 import { RowRendererProps } from './RowRenderer';
 import { getScrollbarSize } from './utils';
 
-const noop = () => {};
+const noop = () => { };
 
 type SummaryRowRendererProps<R, K extends keyof R> = Pick<RowRendererProps<R, K>,
 | 'idx'
 | 'rowData'
 | 'columnMetrics'
-| 'cellMetaData'
 | 'colOverscanEndIdx'
 | 'colOverscanStartIdx'
 | 'rowHeight'
 | 'scrollLeft'
+| 'eventBus'
 >;
 
 function RowRenderer<R, K extends keyof R>({
-  cellMetaData,
   colOverscanEndIdx,
   colOverscanStartIdx,
   columnMetrics,
   idx,
   rowData,
   rowHeight,
-  scrollLeft
+  scrollLeft,
+  eventBus
 }: SummaryRowRendererProps<R, K>) {
   return (
     <Row<R>
```

Comment:
Do we need this component? It does not have any logic and just forward the props